apps/emqx/src/emqx_schema.erl

```diff
@@ -2019,9 +2019,9 @@ common_ssl_opts_schema(Defaults, Type) ->
             )},
         {"cacerts",
             sc(
-                boolean(),
+                hoconsc:array(binary()),
```

Comment:
I think this'll break [this part](https://github.com/id/emqx/blob/7b6379983de845d3d179e8dd2cc2eef8bdc73ec4/apps/emqx_bridge_azure_event_hub/src/emqx_bridge_azure_event_hub.erl#L203).

When loaded via [`public_key:cacerts_get()`](https://github.com/id/emqx/blob/7b6379983de845d3d179e8dd2cc2eef8bdc73ec4/apps/emqx/src/emqx_tls_lib.erl#L671), it's not a list of binaries, it's a bunch of decoded triplets like `{cert, binary(), #OTPCertificate{}}` .

Reply:
[cacerts option in ssl:connect](https://www.erlang.org/doc/man/ssl.html#type-client_cacerts) can be both list of binaries, and list of `#cert{}` records: https://www.erlang.org/doc/man/public_key.html#type-combined_cert

Reply:
Thanks for highlighting this, I'll follow up later.

Reply:
Right, so maybe we could make this an union type then?  The idea of having the boolean value was to have `public_key:cacerts_get()` called if it is true, rather than defining it in the config.

It seems like from the etcd issue that we didn't call `emqx_tls_lib:to_client_opts`?   It should have been filtered there if it's false. :thinking:

Reply:
I think we should just remove this option alltogether and let users specify CAs in the file.

Reply:
Makes sense.  A separate option that injects `cacerts` sounds better.

We would still need to filter such `use_system_cacerts` from the client/server TLS options.  :thinking:

Reply:
Or put it on emqx level, enable by default, and call https://www.erlang.org/doc/man/public_key.html#cacerts_load-0.

Reply:
Removing this option: https://github.com/emqx/emqx/pull/11372